src/gstairplaysrc.c

```diff
@@ -389,7 +407,11 @@ gst_airplay_src_class_init (GstAirPlaySrcClass * klass)
   prop_connected = g_param_spec_boolean ("connected", "Connected",
       "Whether a client is connected to the airplay server",
       FALSE, G_PARAM_READABLE);
+  prop_dnssd_name =  g_param_spec_string ("dnssd-name", "DNS-SD name",
+      "Name to use for the DNS Service Discovery", DEFAULT_DNSSD_NAME,
+      G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
```

Comment:
You need to add `G_PARAM_CONSTRUCT` so that the property is set when the object is created.